pkg/model/iam/iam_builder.go

```diff
@@ -218,7 +222,11 @@ func (b *PolicyBuilder) BuildAWSPolicyNode() (*Policy, error) {
 	}
 
 	if b.Cluster.Spec.Networking != nil && b.Cluster.Spec.Networking.AmazonVPC != nil {
-		addAmazonVPCCNIPermissions(p, resource, b.Cluster.Spec.IAM.Legacy, b.Cluster.GetName())
+		addLyftVPCPermissions(p, resource, b.Cluster.Spec.IAM.Legacy, b.Cluster.GetName())
```

Comment:
This is for AmazonVPC CNI, so I don't think you intended this change (?)

Reply:
ah that was a bit aggressive thanks for catching that